package.json

```diff
@@ -110,6 +118,7 @@
     "build-for-devtools": "cross-env RELEASE_CHANNEL=experimental yarn build react/index,react/jsx,react-dom,react-is,react-debug-tools,scheduler,react-test-renderer,react-refresh --type=NODE && rm -rf build2 && mkdir build2 && cp -r ./build/node_modules build2/oss-experimental/",
     "build-for-devtools-dev": "yarn build-for-devtools --type=NODE_DEV",
     "build-for-devtools-prod": "yarn build-for-devtools --type=NODE_PROD",
+    "improve-images": "node ./scripts/images/improveImages.mjs",
```

Comment:
Can we move this script into [`packages/react-devtools-extensions/package.json`](https://github.com/facebook/react/blob/main/packages/react-devtools-extensions/package.json)? Since it really only affects DevTools images (and almost exclusively ones in the extensions folder?)

Reply:
yes we can, but I think we also should move `script/images/improveImages.mjs` to `packages/react-devtools-extensions/script/images/improveImages.mjs`, or it will be:

```
    "improve-images": "node ../../scripts/images/improveImages.mjs",
```

Reply:
I agree.